tests/test_api.py

```diff
@@ -120,3 +122,92 @@ def test_directory_and_checkout(tmpdir):
     # version than updated
     assert not cruft.check(output_path, strict=True)
     assert cruft.check(output_path, strict=False)
+
+
+@pytest.mark.parametrize(
+    "exit_code,isatty,expect_reproducible_diff,expected_return_value",
+    [
+        (False, False, True, True),  # $ cruft diff | cat
+        (False, True, False, True),  # $ cruft diff
+        (True, False, True, False),  # $ cruft diff --exit-code | cat
+        (True, True, False, False),  # $ cruft diff --exit-code
+    ],
+)
+def test_diff_has_diff(
+    exit_code, isatty, expect_reproducible_diff, expected_return_value, capfd, mocker, tmpdir
+):
+    mocker.patch.object(sys.stdout, "isatty", return_value=isatty)
+
+    project_dir = cruft.create(
+        # FIXME change this to samj1912 repo when it is merged.
+        "https://github.com/gilbsgilbs/cookiecutter-test",
```

Comment:
reminder

Reply:
I also updated these links to point the the right repo. :) Thanks for the PR with the instructions.